en/operations.md

```diff
@@ -18,71 +18,92 @@ The team in charge of an API running in a production environment **SHOULD** ensu
 
 An API **SHOULD** expose an endpoint to check its health status
 
+The parent path ```/health``` **SHOULD** provide some details about components (dependencies)
+
 ```json
 {
-  "name": "Va.Api.Business.MyAwesomeProduct",
-  "status": "up",
-  "dependencies": [
-    {
-      "name": "Va.Api.Tech.Dependency1",
-      "version": null,
-      "depth": 1,
-      "status": "up"
+  "status": "UP",
+  "components": {
+    "discoveryComposite": {
+      "description": "",
+      "status": "UP",
+      "components": {
+        "discoveryClient": {
+          "description": "",
+          "status": "UP"
+        }
+      }
+    },
+    "livenessState": {
+      "status": "UP"
     },
-    {
-      "name": "Va.Api.Tech.SubDependency",
-      "version": "4.9.0-SNAPSHOT",
-      "depth": 2,
-      "status": "up"
+    "readinessState": {
+      "status": "UP"
     }
+  },
+  "groups": [
+    "liveness",
+    "readiness"
   ]
 }
+
+```
+
+The path ```/health/liveness``` **SHOULD** provide the liveness
```

Comment:
Could it be interesting to state the difference between liveness and readiness ?